README.md

```diff
@@ -7,12 +7,12 @@ products:
 description: "Samples showcasing how to used Managed Identity service with Service Fabric."
 urlFragment: service-fabric-managed-identity
 
-author: athinanthny
-ms.author: atsenthi
+author: athinanthny, amenarde
+ms.author: atsenthi, anmenard
 service: service-fabric
 ---
 
-# Code samples using Managed Identity with Service Fabric
+# Getting started with Managed Identity on Service Fabric

```

Comment:
"..for Service Fabric applications"
(We may add support for cluster managed identities.)

Reply:
fixed